src/main/java/uk/gov/digital/ho/pttg/api/ArchiveRequest.java

```diff
@@ -41,9 +45,13 @@ private void validate() {
             throwApiError("A result is required to record in the archive");
         }
 
-        if (eventIds.isEmpty()) {
+        if (correlationIds.isEmpty()) {
             throwApiError("At least one event id is required to be deleted from the audit history");
```

Comment:
If we're changing the name from eventId to correlationId in this class, should the log change similarly?